### PR TITLE
Package rename: focus -> tv.firefox

### PR DESCRIPTION
**Pull request by liuche**
### Pull Request checklist

- [x] This PR includes thorough **tests** or an explanation of why it does not

Package rename, tests run

### CODE REVIEW

File: app/src/main/java/org/mozilla/tv/firefox/generated/LocaleList.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package org.mozilla.tv.firefox.generated;
```
Comment (mcomella) [inline app/src/main/java/org/mozilla/tv/firefox/generated/LocaleList.java]: I think we need to remove org.mozilla.focus.generated (I wonder why this gets generated into the src dir at all though...).

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -39,7 +39,7 @@
                 android:name="android.support.FILE_PROVIDER_PATHS"
                 android:resource="@xml/provider_paths"/>
         </provider>
-        <activity android:name=".MainActivity"
+        <activity android:name="org.mozilla.tv.firefox.MainActivity"
```
Comment (mcomella) [inline app/src/main/AndroidManifest.xml]: I think this can be `.MainActivity`, right?

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -26,7 +26,7 @@
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
-        android:name=".FocusApplication"
```
Comment (mcomella) [inline app/src/main/AndroidManifest.xml]: I think `.FirefoxApplication`, right?

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -76,13 +76,13 @@
             </intent-filter>
         </activity>
 
-        <activity android:name=".browser.InfoActivity"
+        <activity android:name="org.mozilla.tv.firefox.browser.InfoActivity"
```
Comment (mcomella) [inline app/src/main/AndroidManifest.xml]: I don't think these need to change either.

**Comment by liuche** [comment]
Merging this because the TC failure is in fetching resources, and it might be bc of github outages, but local lint passes and this is not important enough to block our hackday.

**Pull request merged**